app/js/lib/form_validation.js

```diff
@@ -97,6 +100,10 @@ var jQuery = require( 'jquery' ),
 				};
 				validationUrl = this.validationUrlForNonSepa;
 			}
+			// avoid sending incomplete data to the server
+			if ( _.find( data, isEmptyString ) !== undefined ) {
```

Comment:
It took me a good number of seconds to fully interpret what this is doing, promoting me to create https://github.com/wmde/FundraisingFrontend/pull/591/commits/f6c2b0298ee96b83292b59116ebca81efa523740. As it is here, you need to deal with not relevant low level details such as the behaviour of the _.find function.

As I found out there, there is no need for such a general check, or for the local variables in this method. https://github.com/wmde/FundraisingFrontend/pull/591/commits/646dcc922e0ff9b4f0c974a2c6868ed27c4c299b

PR: https://github.com/wmde/FundraisingFrontend/pull/591